apps/chat/src/components/Common/DropdownMenu.tsx

```diff
@@ -104,6 +105,13 @@ export const MenuComponent = forwardRef<
   forwardedRef,
 ) {
   const [isOpen, setIsOpen] = useState(isMenuOpen);
+  const openChanged = useCallback(
```

Comment:
```suggestion
  const handleOpenChange = useCallback(
```

lets rename this methid